tests/e2e/bin/build.js

```diff
@@ -0,0 +1,163 @@
+/**
```

Comment:
How would you feel about giving every package a `prepublish` command or such and using `lerna run prepublish` to build all of the packages? I think that might be more accommodating of different package build processes and distribution structures.

Reply:
Could this be postponed until it's needed?

Reply:
I suppose! I'll look at making the change in my pull request for the `factories` package since it has its own build pipeline using TypeScript.